valero_receive_data_from_arduino.m

```diff
@@ -6,12 +6,19 @@
 %txt file.Each line in the text file represents a reading from the 8 
 %strain gauges, and data from each strain gauge is separated by a comma. 
 
-
 %%
+try                     %this is used to close the remaining portal and files

```

Comment:
When I run the code with the proposed changes I get the following error message: 
Error: File:
valero_receive_data_from_arduino.m Line:
1 Column: 1
Invalid use of operator.